include/extractor/extraction_way.hpp

```diff
@@ -46,6 +46,7 @@ struct ExtractionWay
         roundabout = false;
         circular = false;
         is_startpoint = true;
+        ignore_in_grid = false;
```

Comment:
I added this field. Sol was erroring in places where we set this property in [bicycle/foot profiles](https://github.com/Project-OSRM/osrm-backend/blob/0e6863aec1eb1d60d9318cb166a66cd0c20f23e3/profiles/bicycle.lua#L242) but hadn't registered a corresponding field in the bindings. I'm not sure how luabind handled this before.